static/scss/product-details.scss

```diff
@@ -0,0 +1,260 @@
+.container {
+  max-width: 1260px;
+  padding: 0 15px;
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+.hero-block {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  line-height: 30px;
+  color: $black;
+  padding: 20px 0;
+
+  @media (min-width: 768px) {
+    font-size: 24px;
+    line-height: 36px;
+  }
+
+  @media (min-width: 1024px) {
+    padding: 38px 0;
+  }
+
+  .content-row {
+    @media (min-width: 1024px) {
+      display: flex;
+      flex-flow: row wrap;
+      margin: 0 -15px;
+    }
+
+    .content-col {
+      padding: 0 0 20px;
+      @media (min-width: 768px) {
+        padding: 0 0 30px;
+      }
+      @media (min-width: 1024px) {
+        width: calc(100% - 455px);
+        padding: 0 15px;
+        box-sizing: border-box;
+      }
+
+      &.video {
+        padding-bottom: 0;
+        @media (min-width: 1024px) {
+          width: 455px;
+          padding: 0 15px;
+          flex-shrink: 0;
+        }
+      }
+    }
+  }
+
+  .video-holder {
+    background: $sirocco;
+    margin-top: 10px;
+
+    img {
+      display: block;
+      width: 100%;
+    }
+  }
+
+  .text-info {
+    @media (min-width: 1024px) {
+      max-width: 625px;
+    }
+  }
+
+  h1 {
```

Comment:
We should try to avoid specialized styles for different header elements if we can. Does this really need a different font-size, line-height, or font-weight definition from the base styles that we have defined for `h1`? As far as I can tell, this is a header that should look like all other headers of the same size/type